apps/src/maze/MazeVisualizationColumn.jsx

```diff
@@ -0,0 +1,29 @@
+var msg = require('../locale');
+var commonStyles = require('../commonStyles');
+
+var Visualization = require('./Visualization');
+var StepButton = require('./StepButton');
+var SpellingControls = require('./SpellingControls');
+var GameButtons = require('../templates/GameButtons');
+var BelowVisualization = require('../templates/BelowVisualization');
+
+var MazeVisualizationColumn = function (props) {
+  return (
+    <span>
+      <Visualization/>
+      <GameButtons hideRunButton={props.hideRunButton}>
+        <StepButton showStepButton={props.showStepButton}/>
```

Comment:
I agree. However, there's also value in keeping things as similar to the ejs as possible to start with, as it reduces the likelihood of regressions. I think it might make sense to leave things as they are for now, and then after I've converted the visualization column to react for all apps, perhaps do a cleanup pass.